tests/cases/employee_health_portal/case_employee_health_portal_call.yaml

```diff
@@ -184,4 +184,21 @@ TestEmployeeHealthPortalCallsFromCurrentSessionDuplicatedInCallHistoryUntilAcces
       Asserts:
         - Type: eq
           Var: SECOND_ROW_COUNT
-          Value: "1"
\ No newline at end of file
+          Value: "1"
+
+#CXTB-678 Employee Health Portal - Review Call History
+TestEmployeeHealthPortalReviewCallHistory:
+  Environment: Settings
+  Roles:
+    - Role: desktopChrome
+      App: "desktop"
+  Actions:
+  # Calling EHP Login case.
+    - Type: case
+      Value: EmployeeHealthPortalLogIn
+  # Go to Call History
+    - Type: case
+      Value: NavigateToCallHistory
```

Comment:
Can you please add an assert to the end of the test? Do it on the test not the helper navigation so that it only runs on this one